website/public/locales/en/message.json

```diff
@@ -18,6 +18,7 @@
   "report_placeholder": "Why should this message be reviewed?",
   "report_title": "Report",
   "send_report": "Send",
+  "show_image": "Click to show the image",
```

Comment:
I'm not sure about the message, feel free to suggest.

Reply:
That's fine, but maybe we should add some kind of warning: "Warning! Loading from external sources may cause damage."